mods/d2k/rules/structures.yaml

```diff
@@ -596,7 +596,6 @@ outpost:
 		RequiresCondition: !severe-damaged
 	GrantConditionOnDamageState@STOPDISH:
 		Condition: severe-damaged
-		ValidDamageState: Medium, Heavy, Critical
```

Comment:
The correct yaml name here would be `ValidDamageStates`, but this then produces the wrong behaviour ingame.  We don't want to stop the dish on `Medium`, only `Heavy` and `Critical`, which is already the default.